pkg/kubelet/kubelet_pods_test.go

```diff
@@ -5376,6 +5377,9 @@ func TestConvertToAPIContainerStatusesForResources(t *testing.T) {
 			podStatus := testPodStatus(state, resources)
 
 			cStatuses := kubelet.convertToAPIContainerStatuses(tPod, podStatus, []v1.ContainerStatus{tc.OldStatus}, tPod.Spec.Containers, false, false)
+			actual := cStatuses[0]
+			assert.Equal(t, tc.Expected.AllocatedResources, actual.AllocatedResources, "AllocatedResources")
+			assert.Equal(t, tc.Expected.Resources, actual.Resources, "Resources")
```

Comment:
nit: What are these lines for? Looks like the comparison should be covered in L5383 below?

Reply:
It's redundant from a correctness perspective, but much easier to debug. The output of 5383 is basically "this large json blob does not match this other large json blob". Here we're focusing on the parts being intentionally tested, while the other is a catch-all incase an unexpected change was made. I'll add a comment to this effect.